about.html

```diff
@@ -0,0 +1,137 @@
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/ce6302c6e5.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>About</title>
+  </head>
+  <body>
+     <header class="header"> 
+      <ul>
+        <li><a href="#" class="fa fa-twitter"></a> </li>
+        <li><a href="#" class="fa fa-facebook"></a> </li>
+
+        <li>English</li>
+        <li>My page</li>
+        <li>Layout</li>
+        
+      </ul>
+    </header>
+    <nav class="navbar-Mobile">
+      <div class="logonav">
+    <a href="about.html">
+        <img src="images/logo.avif" class="logonav" alt=""></a>
+      </div>
+     <ul class="menu">
+       <li><a href=""><img class="closebtn" src="images/AppBar.png" alt=""></a></li>
+       <li><a href="index.html">About</a></li>
+       <li><a href="">program</a></li>
+       <li><a href="">News</a></li>
+       <li><a href="">Join</a></li>
+       <li><a href="">Sponsor</a></li>
+       <li><a href="">Compaign</a></li>
+     </ul>
+          <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        </div>
+    </nav>
+    <div class="wrapper">
+        
+      <div class="intro">
+        
+        <h3 class="title-about">"Hello Freshers!"</h3>
+        <h1 class="about-h1">Mobile first design strategy</h1>
+        <p class="about-info">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit
+          magnam odit praesentium excepturi eveniet, dolor dolorem ex incidunt
+          perferendis
+        </p>
+        <p class="caption-about">2020/30/10 Join us for better knowledge</p>
+        <p class="caption-about1">@Kabul University Afghanistan</p>
+      </div>
+      <div class="main-program">
+        
+        <h2 class="main-program1">Main program</h2>
+        <div class="main">
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Lecture</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">UI Exhibition</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Forum</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur.</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Workshop</h3>
+          <p class="lecture-p">Lorem ipsum dolor</p>
+        </div>
+        <div class="lecturesection">
```

Comment:
- [ ] `required`
- [ ] Kindly add icons like in the provided design.
- [ ] see screenshot belows
`your design`
![2page](https://user-images.githubusercontent.com/66230306/190278845-f1d61c66-9bce-4d36-8213-4586d0cdad3d.png)

- [ ] `Provided design`

![image](https://user-images.githubusercontent.com/66230306/190279106-c1f0eb53-8ad9-450b-bb04-002f9eb035fa.png)